pkg/bootstrapper/bootstrapper_test.go

```diff
@@ -551,3 +551,20 @@ func testCNIUpdateKubeletArgs(t *testing.T) {
 		assert.Contains(t, err.Error(), "nil kubelet cmd passed")
 	})
 }
+
+// TestPodManifestDirCreation tests if the pod manifest directory was created
+func TestPodManifestDirCreation(t *testing.T) {
+	podManifestDirectory := "c:\\k\\etc\\kubernetes\\manifests"
```

Comment:
nit: a seperate commit that moves c:\k to a global variable would be cool, but not needed